src/transformers/models/cpmant/configuration_cpmant.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+# Copyright 2022 The OpenBMB Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" CPMAnt model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+CPMANT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "cpm-ant-10b": "https://huggingface.co/cpm-ant-10b/resolve/main/config.json",
```

Comment:
This checkpoint seems to be not public, can you please double check?, maybe you mean
`https://huggingface.co/openbmb/cpm-ant-10b/blob/main/config.json`

Reply:
Thanks for the reminder! The ckpt has been public in https://huggingface.co/openbmb/cpm-ant-10b